<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckItem</class>
 <widget class="QWidget" name="CheckItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>225</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>225</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CheckItem{
background: #122033;
border-radius: 0px;
border:1px solid #122033;
border-left:1px solid #4E5969;
border-right:1px solid #4E5969;
}
QLineEdit{
padding-left:10px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget
{
background:transparent;
border:none;
}
QLabel{
color:#1a1a1a;
font-size:16px;
line-height:24px;
}

QLineEdit{
color:#12D94D;
border:none;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;

}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #12D94D;
border:none;</string>
        </property>
        <property name="text">
         <string>编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_num">
        <property name="text">
         <string>JS20240114</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget
{
background:transparent;
border:none;
}
QLabel{
color:#1a1a1a;
font-size:16px;
line-height:24px;
}

QLineEdit{
color: #FF36F2;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;

}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
color: #FF36F2;</string>
        </property>
        <property name="text">
         <string>医师：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_user">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>张飞</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget
{
background:transparent;
border:none;
}
QLabel{
color:#1a1a1a;
font-size:16px;
line-height:24px;
}

QLineEdit{
color: #1479FF;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;

}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1479FF;</string>
        </property>
        <property name="text">
         <string>项目：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_project">
        <property name="text">
         <string>胃肠镜</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget
{
background:transparent;
border:none;
}
QLabel{
color: #FC8217;
font-size:16px;
line-height:24px;
}

QLineEdit{
color: #FC8217;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;

}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_date">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>2024-01-14 19:25:10</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
